src/SOS/SOS.NETCore/SymbolReader.cs

```diff
@@ -332,6 +332,9 @@ public static string GetSymbolFile(SymbolStoreKey key)
                         {
                             downloadFilePath = file.FileName;
 
+                            // Make sure the stream is at the beginning of the module
+                            file.Stream.Position = 0;
```

Comment:
This seems surprising, I would have expected GetSymbolStoreFile() to put the stream at zero before returning it.

Reply:
Yes, it should and I will fix it, but for now I'm working around it here.  It is the new "directory" search symbol store. The http one doesn't have this problem.

Reply:
The symstore issue is tracked under https://github.com/dotnet/symstore/issues/117